autotest/test_ssaver_loop.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# test_ssaver_loop.sh
+#
+# Simple rustic script to automate Kano OS screen saver testing.
+# It will go into the following cycle in an endless loop.
+#
+# Dashboard -> Overworld -> SSaver -> Dashboard -> Ssaver -> Classic Mode -> SSaver, back to start.
+#
+# Requirements:
+#
+#  1) edit /usr/share/kano-desktop/kdesk/kdeskrc_screensaver, set ScreenSaverTimeout: 15
+#  2) edit /usr/share/kano-desktop/kdesk/.kdeskrc, set ScreenSaverTimeout: 15
+#  3) Make sure you are on the Dashboard on idle status, then run:
+#
+#  $./ssaver-loop.sh > $HOME/ssaver-loop.log 2>&1
```

Comment:
The name of the script is wrong, should be `./test_ssaver_loop.sh` :+1:

Reply:
Thanks Ioannis! Fixed